source/guides/guides/network-requests.md

```diff
@@ -318,6 +320,47 @@ cy.get('#results')
 - Response Body
 - Response Headers
 
+**Examples**
+
+```javascript
+cy.server()
+// spy on POST requests to /users endpoint
+cy.route('POST', '/users').as('new-user')
+// trigger network calls by manipulating web app's user interface, then
+cy.wait('@new-user')
+  .should('have.property', 'status', 201)
+// we can grab the XHR object again to run more assertions
+cy.wait('@new-user') // yields XHR object
+  .its('request.body')
+  .should('deep.equal', {
+    id: '101',
+    firstName: 'Joe',
+    lastName: 'Black'
+  })
+// and we can place multiple assertions in a single "should" callback
+cy.wait('@new-user')
```

Comment:
also `cy.get('@new-user')`? I'm not toally familiar with whether it's suppose to be 3 POSTs or just one your asserting on since the "we can grab the XHR object again" comment.